### ISSUE TITLE
How to keep message in `test results`

### ISSUE THREAD

**Issue by fly50789** [jest-community/vscode-jest#1090]
### Environment

1. `vscode-jest version`: [fill] v5.2.3
2. `node -v`: [fill] v18.16.1
3. `npm -v` or `yarn --version`: [fill]9.5.1
4. `npm ls jest` or `npm ls react-scripts` (if you haven’t ejected): [fill]
```
PS C:\Users\Pega_User\Desktop\project\temp\node_js> npm ls jest
node_js@1.0.0 C:\Users\Pega_User\Desktop\project\temp\node_js
└── jest@29.7.0

```
6. your vscode-jest settings if customized: (no change)
    - jest.jestCommandLine? [fill]
    - jest.autoRun? [fill]
    - anything else that you think might be relevant? [fill]
7. Operating system: [fill] windows 10

### Prerequisite
- are you able to run jest test from the command line? [fill] yes
- how do you run your tests from the command line? (for example: `npm run test` or `node_modules/.bin/jest`) [fill]
![image](https://github.com/jest-community/vscode-jest/assets/17740390/062bcdae-db77-4312-aa2d-26e60a5db710)

### Steps to Reproduce

[fill]

### Relevant Debug Info

[fill]

### Expected Behavior

log should be keep in  TEST RESULTS, but it disappear after test.

### Actual Behavior

when running, it continue output message.

But after test finish, message is disappear

<img width="1007" alt="image" src="https://github.com/jest-community/vscode-jest/assets/17740390/dd4fc9f9-599e-4f58-b36b-709feda665bc">

![image](https://github.com/jest-community/vscode-jest/assets/17740390/3c9f2ddb-ee48-4021-b542-c6fa24920c76)

--- 

The fastest (and the most fun) way to resolve the issue is to submit a pull-request yourself. If you are interested, feel free to check out the [contribution guide](https://github.com/jest-community/vscode-jest/CONTRIBUTING.md), we look forward to seeing your PR...

**Comment by connectdotz** [comment]
Hi, this should be fixed in [v6.1.0 pre-release](https://github.com/jest-community/vscode-jest/blob/master/release-notes/release-note-v6.md#v610-pre-release). Feel free to give it a try.

**Comment by fly50789** [comment]
@connectdotz  It just work!! Thank you!!

![image](https://github.com/jest-community/vscode-jest/assets/17740390/4fd5f199-e9fb-4d25-9ebd-c0874ec8d8a0)
